WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -256,10 +256,12 @@ import class WooFoundation.MockAnalyticsProviderPreview
                                           cardPresentPaymentService: CardPresentPaymentPreviewService(),
                                           currencyFormatter: .init(currencySettings: .init()),
                                           paymentState: .acceptingCard)
-    let cartViewModel = CartViewModel(posModel: PointOfSaleAggregateModel(itemProvider: POSItemProviderPreview()))
-    let itemsListViewModel = ItemListViewModel(posModel: PointOfSaleAggregateModel(itemProvider: POSItemProviderPreview()))
-    let dashboardViewModel = PointOfSaleDashboardViewModel(posModel: PointOfSaleAggregateModel(itemProvider: POSItemProviderPreview()),
-                                                           cardPresentPaymentService: CardPresentPaymentPreviewService(),
+    let cartViewModel = CartViewModel(posModel: PointOfSaleAggregateModel(itemProvider: POSItemProviderPreview(),
+                                                                          cardPresentPaymentService: CardPresentPaymentPreviewService()))
+    let itemsListViewModel = ItemListViewModel(posModel: PointOfSaleAggregateModel(itemProvider: POSItemProviderPreview(),
+                                                                                   cardPresentPaymentService: CardPresentPaymentPreviewService()))
+    let dashboardViewModel = PointOfSaleDashboardViewModel(posModel: PointOfSaleAggregateModel(itemProvider: POSItemProviderPreview(),
+                                                                                               cardPresentPaymentService: CardPresentPaymentPreviewService()),
```

Comment:
We're updating this one everywhere in our previews, should we create a static `PointOfSaleAggregateModel.preview()` helper?

Reply:
Thanks for the suggestion, yeah it might help reduce this pain. 

By the time I'm done, I hope it won't be needed so for now I'm just making it work...